File: backend/src/test/java/modelo/UsuarioTest.java
package modelo;

import Elementos.Ing.AgileTasks.modelo.Usuario;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class UsuarioTest {

    Usuario usuario = new Usuario();

    @Test
    public void usuarioTest(){
        usuario.setPassword("1234");
        usuario.setUserName("admin");

        assertEquals(usuario.getUserName(), "admin");
        assertEquals(usuario.getPassword(), "1234");
    }
}

File: backend/src/test/java/modelo/UsuarioTest.kt
package modelo

import Elementos.Ing.AgileTasks.modelo.Usuario
import org.junit.Assert
import org.junit.jupiter.api.Test

class UsuarioTest {

    var usuario : Usuario = Usuario()

    @Test
    fun usuarioTest(){
        usuario.password = "1234"
        usuario.userName = "admin"

        Assert.assertEquals(usuario.userName, "admin")
        Assert.assertEquals(usuario.password, "1234")
    }
}